diagnostic.md

```diff
@@ -44,9 +57,23 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
+```CSS
+.big{
 
-<!-- your answer ends here -->
+}
+
+#contentPane{
+
+}
+
+.important .red{
+
+}
+
+div p{
```

Comment:
```div p``` will select all ```<p>```s that have ```<div>``` as an ancestor regardless of depth.  ```div > p``` selects ```<p>```s that have a ```<div>``` as a parent.